src/java.base/share/classes/java/lang/invoke/ClassSpecializer.java

```diff
@@ -64,11 +65,15 @@
  * @param <K> key which identifies individual specializations.
  * @param <S> species data type.
  */
+@AOTSafeClassInitializer
```

Comment:
This should be placed below the `/*non-public*/` comment.

Reply:
Perhaps also the same for BoundMethodHandle?

Reply:
John disclosed these comments were from the initial iteration of JSR 292 implementation. Newer code does not have this comment, so I think I will just remove them outright.